recipes/stir/meta.yaml

```diff
@@ -33,14 +33,18 @@ requirements:
     - numpy >=1.11  # [win]
 
 test:
+  requires:
+    - pytest
   imports:
     - stir
   source_files:
     - recon_test_pack
+    - src/swig/test/python
   commands:
     - conda inspect linkages -p $PREFIX {{ name|lower }}  # [not win]
     - conda inspect objects -p $PREFIX {{ name|lower }}   # [osx]
     - cd recon_test_pack && ./run_tests.sh --nointbp  # [not win]
```

Comment:
win has a `bat` file.
linux has quite a few other .sh (you could check the `.travis.yml`). clearly, we need to integrate those all in one script in the STIR side.

Reply:
I'm using the bat file as well now, so proper tests are also run on Windows. I think that perhaps suffices as far as tests are concerned? (interpreting [conda-forge's documentation](https://conda-forge.org/docs/testing.html#should-a-recipe-run-all-of-a-package-s-tests))